packages/api/src/domain/facility-mapping.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseDomain } from "@metriport/core/domain/base-domain";
+
+export interface FacilityMapping extends BaseDomain {
+  externalId: string;
+  cxId: string;
+  facilityId: string;
+  source: string;
```

Comment:
We don't have a type for the list of EHRs?

Reply:
This table is more general than that but I can create a mapping source type that's only the EHR sources 👍

Reply:
Done.